vowpalwabbit/vw.h

```diff
@@ -115,10 +115,11 @@ void dealloc_example(void (*delete_label)(polylabel*), example& ec, void (*delet
 void dealloc_examples(example* example_ptr, size_t count);
 
 void parse_example_label(vw& all, example& ec, std::string label);
-void setup_examples(vw& all, v_array<example*>& examples);
+void setup_examples(vw& all, std::vector<example*>& examples);
```

Comment:
since we intend for this to eventually go into master, we need to be extremely careful when we're modifying this file. These vector changes breaks the API, which is probably not ok. The correct way to handle this would be to create new entry points, keep (but deprecate) the old interfaces, then write a translation layer between the two